### DISCUSSION TITLE
Student Document is not accepted

### DISCUSSION THREAD

**Discussion by turret001** [community/community#44112] [GitHub Education] [answered]
Hi,
In Turkey, universities don't issue student papers anymore, you have to get your proof of education from the government portal edevlet. 
I've been trying to upload my papers to Github (as PNG) but system keeps rejecting it and telling me to get papers from the faculty. 

I'm not sure what to do, the document I'm uploading has barcode proof and everything (which can be used to verify authenticity)

**Comment by melvincwng** [comment, marked as answer]
Hey @turret001 👋🏻, I assume you are applying for the GitHub Student Developer pack. If that's the case, continue reading this message.

As quoted from the [GitHub Docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/apply-to-github-global-campus-as-a-student#requirements):
`Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.`

Do you happen to have any of those documents as stated above? If yes, you can use any of those as proof of being a student during the submission to GitHub.

If you have multiple documents that can prove your status of being a student, first try submitting one of them first. If it still fails, try uploading an alternative document. 

If all fails, the best solution would be to directly contact [GitHub Education Support](https://support.github.com/contact/education).

Hope this clarifies and helps answer your question. 

Take care, and have a good day ahead 😄

**Reply by turret001** [reply]
Thank you, Melvin, I've tried re-submitting my papers but the system didn't accept them once again. I better contact support, thanks!

**Reply by melvincwng** [reply]
Glad to have helped 😃

**Comment by samus-aran** [comment]
Hi @turret001 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍

**Comment by jlee0310** [comment]
Hi, 
I'm in online school and wondering how can I prove 
"You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so."

**Reply by samus-aran** [reply]
Hi @jlee0310 👋 

We would recommend creating a new discussion if your question is not related to the current topic at hand. 

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

I would also like to suggest that often re-reviewing the application and supplying more preferred evidence would assist us in affirming your student status. 
Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

**Reply by Dart2112** [reply]
> Hi @jlee0310 👋
> 
> We would recommend creating a new discussion if your question is not related to the current topic at hand.
> 
> For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.
> 
> I would also like to suggest that often re-reviewing the application and supplying more preferred evidence would assist us in affirming your student status. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

Does the application actually get submitted even when this error is given in the pre-check? I am trying to submit proper documentation but it seems like it auto-denies it because my location was wrong once. The website isn't requesting location permission anymore even on different devices.

**Comment by ahmedelshkh** [comment]
Hi,

I'm from Egypt so my Student Id is in Arabic
I upload it and it keeps rejecting me and says I must complete my profile. Just so you know, my profile is already complete.
![Request a discount - GitHub Education - Google Chrome 2_11_2023 11_09_44 PM (2)](https://user-images.githubusercontent.com/53626145/218281382-b1370f29-f361-4b40-9439-4ba79eeb8bde.png)


Is there a way for me to fill in my student id data manually? or what do I have to do?

**Reply by mohamadhesham00** [reply]
I am facing the same issue did you solve yours?

**Reply by ahmedelshkh** [reply]
nope, I think the problem in my university email,
every one of my friends tried had the same issue

**Comment by turret001** [comment]
Hi everyone, when I tried to submit my papers after a while, they were automatically accepted! 
Now, I can't remember whether it has been a week or a month but they got processed automatically.

If you're having the same issue as I've had, you would be better off creating an issue of yours instead of replying to this one (or better yet, check melvin's reply https://github.com/orgs/community/discussions/44112#discussioncomment-4670100 )

good luck!
